src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -112,7 +114,13 @@ public static FontFamily Parse(string s)
 
                 case 2:
                     {
-                        return new FontFamily(segments[1], new Uri(segments[0], UriKind.RelativeOrAbsolute));
+                        var uri = segments[0].StartsWith("/")
```

Comment:
We probably need some kind of Uri parsing helper to be used in converters since `new Uri(string)` misbehaves on Linux by trying to parse `/some/path` as a file Uri. It would be nicer to have that logic in one place.